lib/automaticupgrades/config.go

```diff
@@ -56,3 +61,18 @@ func IsEnabled() bool {
 func GetChannel() string {
 	return os.Getenv(automaticUpgradesChannelEnvar)
 }
+
+// GetUpgraderVersion returns the teleport upgrader version
+func GetUpgraderVersion(ctx context.Context) string {
+	if os.Getenv("TELEPORT_EXT_UPGRADER") == "unit" {
+		out, err := exec.CommandContext(ctx, teleportUpgradeScript, "version").CombinedOutput()
```

Comment:
nit: best to only capture stdout in case we add additional logging in the future.
```suggestion
		out, err := exec.CommandContext(ctx, teleportUpgradeScript, "version").Output()
```